port_ocean/core/event_listener/immediate.py

```diff
@@ -0,0 +1,60 @@
+import signal
+from typing import Literal, Any
+
+from loguru import logger
+
+from port_ocean.core.event_listener.base import (
+    BaseEventListener,
+    EventListenerEvents,
+    EventListenerSettings,
+)
+from port_ocean.utils import repeat_every
+
+
+class ImmediateEventListenerSettings(EventListenerSettings):
+    """
+    Immediate event listener configuration settings.
+    This class inherits from `EventListenerSettings`, which provides a foundation for creating event listener settings.
+    """
+
+    type: Literal["IMMEDIATE"]
+
+    def to_request(self) -> dict[str, Any]:
+        return {}
+
+
+class ImmediateEventListener(BaseEventListener):
```

Comment:
Maybe we should not trigger the on start function
Because some integrations will create webhooks

Reply:
passed the event listener type to the start event context, so that each integration can decide on its behavior based on the event listener type